client/.eslintrc.json

```
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb",
+        "prettier"
```

You may see this eslint error `'React' must be in scope when using JSX react/react-in-jsx-scope`

As we used newer version of react, according to their docs https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md


```suggestion
        "airbnb",
        "plugin:react/jsx-runtime",
        "prettier"
```